libsplinter/src/biome/rest_api/mod.rs

```diff
@@ -62,8 +62,8 @@ use self::actix::key_management::{
 };
 
 #[cfg(feature = "biome-key-management")]
-use super::key_management::store::PostgresKeyStore;
-use super::user::store::diesel::SplinterUserStore;
+use crate::biome::key_management::store::PostgresKeyStore;
```

Comment:
There are still many of these one-level super:: getting changed.

Reply:
Updated to use super. I initially left these as the full paths because at a glance `super::key_management` can be a bit ambiguous because there is `biome::key_management` and `biome::rest_api::actix::key_management`. So what super is referring to might not be immediately obvious to someone who is new to biome.